posts/2020/02/whats-next-for-pydatasparse.md

```diff
@@ -0,0 +1,37 @@
+<!--
+.. title: What's next for PyData/Sparse
```

Comment:
I'm not sure this is "next" - it's not unlikely that this will take a long time and other improvements happen first. How about "Plans for lazy evaluation for PyData/Sparse" or "Planned architectural changes for PyData/Sparse"?

Reply:
Done!